File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-1_14.rb
puts ('Vvedite veshestvennoe chislo: ')
a = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
b = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
c = gets.to_i
c_al = (b ** 2.0 + c ** 2.0 - a ** 2.0) / (2 * b * c)
al = Math.acos(c_al) * 180 / Math::PI
puts ("#{al}")
c_bet = (a ** 2.0 + c ** 2.0 - b ** 2.0) / (2 * a * c)
bet = Math.acos(c_bet) * 180 / Math::PI
puts ("#{bet}")
c_gam = (b ** 2.0 + a ** 2.0 - c ** 2.0) / (2 * b * a)
gam = Math.acos(c_gam) * 180 / Math::PI
puts ("#{gam}")

File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-1_14.go
package main
import ("fmt"
        "math"
)

func main() {
    var a float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&a)
    var b float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&b)
    var c float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&c)
    pi := 3.14159265
    c_al := (math.Pow(b, 2) + math.Pow(c, 2) - math.Pow(a, 2)) / (2 * b * c)
    al := math.Acos(c_al)
    fmt.Println (al * 180 / pi)
    c_bet := (math.Pow(a, 2) + math.Pow(c, 2) - math.Pow(b, 2)) / (2 * a * c)
    bet := math.Acos(c_bet)
    fmt.Println (bet * 180 / pi)
    c_gam := (math.Pow(b, 2) + math.Pow(a, 2) - math.Pow(c, 2)) / (2 * b * a)
    gam := math.Acos(c_gam)
    fmt.Println (gam * 180 / pi)
}